Task 1

SlavikSavenok opened the pull request:
Задачи_1_2_3. 
Задаие_0 в главной ветке

SlavikSavenok committed:
Создал массив [8][8] из букв W & B.

SlavikSavenok committed:
Создал массив [5][5] и вывел сумму элементов.

SlavikSavenok committed:
Создал массив [5][5] и вывел его диагонали.

dmitars reviewed (changes requested):
There are no massiv word, again

dmitars commented on src/Task_3.java:
```diff
@@ -0,0 +1,36 @@
+        for (int i = 0; i < massivForDiagonals.length; i++) {
+            int Diagonal;
+            Diagonal = massivForDiagonals[i][i];
+            System.out.print(Diagonal + " ");
```
Diagonal - why with a big letter?

dmitars commented on src/Task_3.java:
```diff
@@ -0,0 +1,36 @@
+        for (int i = 0; i < massivForDiagonals.length; i++) {
+            for (int j = 0; j < massivForDiagonals.length; j++) {
+                if (j == massivForDiagonals.length - i - 1) {
+                    System.out.print(massivForDiagonals[i][j] + " ");
```
Can we do that using a single cycle?

SlavikSavenok replied:
I don’t know how yet, to maintain the ratio j=n-i-1 you need two cycles.
The only thing that comes to mind so far is to create a new variable and somehow replace the loop with it. If I don’t do it by Tuesday, then this will be a good question at the beginning of the lesson :)

SlavikSavenok replied:
Done

SlavikSavenok committed:
Изменил заглавную букву diagonal в 3-м задании.
Изменил название массива на более полное в 1-м задании (если именно это подразумевалось "There are no massiv word, again"

SlavikSavenok committed:
В 3-м задании вывел диагональ при помощи одного цикла
